data/boosters/fdn-prerelease.yaml

```diff
@@ -0,0 +1,7 @@
+pack:
+  prerelease_promo: 1
+sheets:
+  prerelease_promo:
+    filter: 'e:p{set} promo:prerelease number<=257s'
```

Comment:
Is there a particular reason you included the number range filter? It's not _wrong_ per-se, but it doesn't really do anything since searching for `number>=258` returns no results.

Reply:
The query for either appears to return the same results without any constraint on number.